### ISSUE TITLE
Transfer learning faulty evaluation parameters

### ISSUE THREAD

**Issue by MajaLijes** [MannLabs/alphapeptdeep#276]
Hello @mschwoer! and others from Alphapeptdeep,

I've had a problem with transfer learning data from MS_Fragger (DDA data)

Everything seems find when running the transfer learning function, but when i come to the evaluation parameters of the MS2 model, then it writes the following:

2026-03-19 14:01:33> Testing refined MS2 model on training df:
          PCC     COS      SA          SPC
count  2357.0  2357.0  2357.0  2357.000000
mean      0.0     0.0     0.0     0.578473
std       0.0     0.0     0.0     0.280401
min       0.0     0.0     0.0    -0.530592
25%       0.0     0.0     0.0     0.454899
50%       0.0     0.0     0.0     0.653526
75%       0.0     0.0     0.0     0.780145
max       0.0     0.0     0.0     1.000000
(>)0.90     0.0     0.0     0.0     0.022486
(>)0.75     0.0     0.0     0.0     0.316504
2026-03-19 14:01:33> Finished training MS2 model

Here it is seen that all my parameters PCC, COS and SA are all 0, but funnily enough my SPC has values. When training occurs the learning rate and loss function also seems fine, so i do not understand what goes wrong. It do also create an updated model.

I've inserted 1 PSM file (msfragger_pepxml file) and 1 raw file which fits to the PSM file. Do you guys know why my parameters become 0?

Kind regards
Maja Lindskov Jessen
